addons/test_mass_mailing/tests/test_performance.py

```diff
@@ -46,8 +46,8 @@ def test_send_mailing(self):
             'mailing_domain': [('id', 'in', self.mm_recs.ids)],
         })
 
-        # runbot needs +51 compared to local
-        with self.assertQueryCount(__system__=1524, marketing=1525):
+        # runbot needs +98 compared to local
+        with self.assertQueryCount(__system__=1571, marketing=1572):
```

Comment:
I couldn’t find the reason of the ~50 added queries. Do you have an idea ? It seems not related to the added compute method in link tracker because it is not called during the test. It may be related to “base_url = self.mailing_id.get_base_url()” changed in “base_url = self.get_base_url()”. 
I probably miss something, sorry. 
Maybe it would be worth adding a comment in the commit about that.

Reply:
Actually to be more accurate, it comes from `get_base_url()` called in `_get_unsubscribe_url()`